### ISSUE TITLE
Login spoke by mozilla doesn't work

### ISSUE THREAD

**Issue by Neburgi** [mozilla/Spoke#1280]
I enter in this page https://hubs.mozilla.com/spoke/login
but when I introduce my email, it doesn't work. So I can't enter to my project. 
I don't know how I can enter. I verified the folder spam, all the rules of my email but I can't find a solution

**Comment by matthewbcool** [comment]
Sorry to hear about the trouble. I was not able to reproduce.

I tried:

- logging out of Spoke from https://hubs.mozilla.com/spoke/

- logging back in with my account email.

I received the link in my inbox and logged in as expected. 

Can you confirm the email is correct? Perhaps try verifying with another email address?

**Comment by Neburgi** [comment]
yep. I rtry with another emails and work (hotmail and gmail), but in the other account i have a project

**Comment by Neburgi** [comment]
Also, I used different dispositives, just in case. But doesn't work too.
